README.md

```diff
@@ -49,4 +49,36 @@ pyrsia 0.2.0
 ```
 $ pyrsia ping
 Connection Successful !!
+```
+
+## Installer Release Guide
+
+Following sequence diagram demonstrate what needs to be done after successful release of Pyrsia to update the macOS installer.
+
+```mermaid
+sequenceDiagram
+    title Pyrsia Homebrew Installer Manual Release Process
+    
+    participant p as pyrsia
+    participant gcb as Google Cloud Bucket
+    participant user as User🧍
+    participant hp as homebrew-pyrsia
+    
+    rect rgb(190, 220, 255)
+    note right of p: Release
+    p ->> p: Release Pyrsia
+    p ->> gcb: Github Action - Push Release Archive 
+
+    rect rgb(200, 150, 255)
+    note right of gcb: homebrewrepo/stable/x86_64/<archive_name>
+    note right of gcb: accessible using https://brewrepo.pyrsia.io/stable/x86_64/<archive_name>
+    end
+    end
+    rect rgb(100, 255, 190)
+    user ->> hp: manually update Formula/pyrsia.rb
+    user -->> hp: update archive_name in the url
+    user -->> hp: calculate sha256 and update sha256
+    user -->> hp: update version
```

Comment:
Are dashed lines opposite?

```suggestion
    hp -->> user: update archive_name in the url
    hp -->> user: calculate sha256 and update sha256
    hp -->> user: update version
```
![image](https://user-images.githubusercontent.com/8938191/204879844-25a33587-57b3-4fae-af37-d554f0f0843e.png)

Reply:
here dashed line are subcomponents of what user needs to update manually. 3 elements need to be updated manually i.e. url, sha256, version.

Reply:
@dasmanas Ah okay thank you. I thought users should just `brew update` and `brew upgrade pyrsia`... 😓 
Do we have documents which specific commands and procedures are written?

Reply:
If not, I will add the info.